trick_source/sim_services/DataRecord/DataRecordGroup.cpp

```diff
@@ -692,6 +693,14 @@ int Trick::DataRecordGroup::write_data(bool must_write) {
             writer_num++ ;
 
         }
+
+        if(!max_size_warning && (total_bytes_written > max_file_size)) {
+            std::cerr << "WARNING: Data record max file size " << (max_file_size>>10) << "KB reached. Contact Derek Bankieris regarding any concerns.\n"
```

Comment:
Nice try, bucko! Also, kilobytes? Nobody wants to see a number that big! Also also, shifting will result in a value of 0 if they set the limit below 1 KB. Better use real division! Also also also, look at this cool new suggestion feature!
```suggestion
            std::cerr << "WARNING: Data record max file size " << (static_cast<double>(max_file_size)/(1<<20) << "MB reached.\n"
```